Antistasi-Ultimate-Community/A3-Antistasi-Ultimate
# Stalker templates

## What type of PR is this.
1. [ ] Bug
2. [x] Change
3. [x] Enhancement

### What have you changed and why?
Information:
Added STALKER templates/STALKER themed stuff because stalker anomaly got me intrigued

### Please specify which Issue this PR Resolves.
closes #XXXX

### Please verify the following and ensure all checks are completed.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?
1. [ ] No
2. [x] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps: 
Make sure civ spawning still works, unless on zombie/no civ template (done this locally)
Ensure anomalies/emissions don't majorly fuck up mission balance (done this locally)
Make sure the factions all work (done this locally)

********************************************************
Notes:
